custom/lib/Auth.php

```diff
@@ -130,10 +128,8 @@ public function createMagicLink(
             }
         }
 
-        $magicLinksApi = new MagicLinksApi(null, $this->config);
-
```

Comment:
Curious: Is this personal preference to move this? Does it support a different purpose?

Reply:
a little of both. the move to the constructor ensures the object only gets put on the heap once. otherwise it gets created on every invocation